definitions/features/pulp2.rb

```diff
@@ -5,7 +5,7 @@ class Features::Pulp < ForemanMaintain::Feature
     label :pulp2
 
     confine do
-      find_package('pulp-server')
+      find_package('pulp-server') && check_max_version('katello-common', '3.18')
```

Comment:
I wonder how this works with `katello-common-3.18.1`. Sadly `check_max_version` has no API documentation and no tests.

Reply:
We could add a test or two here for that scenario. It's a good catch.

Reply:
It doesn't appear that check_max_version will work here.

`check_max_version('katello-common', '3.18')` returns false if the current `katello-common` version is `3.18.1`, etc.